samwise.js

```diff
@@ -2,7 +2,7 @@
  * Autoplay plugin - Automatically advance slideshow after N seconds
```

Comment:
SCRIPT-Consider using more descriptive naming for this boolean variable to clarify its purpose.